src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -97,8 +97,8 @@ public void removeEatery(Eatery key) {
 
     @Override
     public String toString() {
-        return eateries.asUnmodifiableObservableList() + " eateries";
-        // TODO: refine later
+        return String.format("%d eateries: %s", eateries.asUnmodifiableObservableList().size(),
+                eateries.asUnmodifiableObservableList());
```

Comment:
good that you considered checking both the size and the eateries

Reply:
can u shut up